elliottlib/cli/find_bugs_cli.py

```diff
@@ -208,8 +215,9 @@ def find_bugs_cli(runtime: Runtime, advisory, default_advisory_type, mode, check
         if mode != 'qe' and exclude_status:
             status = set(status) - set(exclude_status)
 
-        green_prefix(f"Searching for bugs with status {' '.join(status)} and target release(s):")
-        click.echo(" {tr}".format(tr=", ".join(bz_data['target_release'])))
+        if output == 'text':
```

Comment:
These ideally should be logger.info, suppressed by --quiet. Would leave it to you, if you want to do that here or not

Reply:
I'd rather not, this is a issue we might want to tackle but I would do it in a separate task. Would leave as-is for now